symbolic-il2cpp/src/usym.rs

```diff
@@ -610,6 +610,21 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_header_with_errors() {
+        let file = File::open(fixture("il2cpp/artificial-bad-meta.usym")).unwrap();
+        let data = ByteView::map_file_ref(&file).unwrap();
```

Comment:
```suggestion
        let data = ByteView::open(fixture("il2cpp/artificial-bad-meta.usym")).unwrap();
```

;-)

Reply:
oh, thanks for this! this goes to show up much copy-pasting i've been doing 🙃